sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -516,19 +514,15 @@ def _setup_transport(self):
             self.sock = self._wrap_socket(self.sock, **self.sslopts)
         self._quick_recv = self.sock.recv
 
-    def _wrap_socket(self, sock, context=None, **sslopts):
-        if context:
-            return self._wrap_context(sock, sslopts, **context)
-        return self._wrap_socket_sni(sock, **sslopts)
-
-    def _wrap_context(
-        self, sock, sslopts, check_hostname=None, **ctx_options
+    def _wrap_socket(
+        self,
+        sock,
+        **sslopts
     ):
-        ctx = ssl.create_default_context(**ctx_options)
```

Comment:
if they dont specify a ssl context or a verify_location ( the current default ) wont this block be needed?

Reply:
if they don't pass in a context, this will go in into _wrap_socket_sni which will create an ssl context object